internal/services/netapp/netapp_volume_resource.go

```diff
@@ -561,7 +556,7 @@ func resourceNetAppVolumeDelete(d *pluginsdk.ResourceData, meta interface{}) err
 
 			// Waiting for replication be in broken state
 			log.Printf("[DEBUG] Waiting for replication on NetApp Volume Provisioning Service %q (Resource Group %q) to be in broken state", replVolumeID.Name, replVolumeID.ResourceGroup)
```

Comment:
`log.Printf("[DEBUG] Waiting for replication on NetApp Volume Provisioning Service %s to be in broken state", replVolumeID)`